src/pages/en/guides/migrate-to-astro/from-nextjs.mdx

```diff
@@ -2,72 +2,924 @@
+
+```astro title="src/components/Link.astro"
+---
+const { to } = Astro.props
+---
+<a href={to}><slot /></a>
+```
+
+### Next Imports to Astro
+
+If necessary, update any [file imports](/en/guides/imports/) to reference relative file paths exactly. This can be done using [import aliases](/en/guides/typescript/#import-aliases), or by writing out a relative path in full. 
+
+Note that `.astro` and several other file types must be imported with their full file extension.
+
+```astro title="src/pages/authors/Fred.astro"
+---
+import Card from `../../components/Card.astro`
+---
+<Card />
+```
+
+### Next Children Props to Astro
+
+Convert any instances of `{children}` to an Astro `<slot />`. Astro does not need to receive `{children}` as a function prop and will automatically render child content in a `<slot />`.
+
+```astro title="src/components/MyComponent" del={3-9} ins={11-13}
+---
+---
+export default function MyComponent(props) { 
+    return (
+      <div>
+        {props.children}
+      </div>
+    );  
+}
+
+<div>
+  <slot />
+</div>
+```
+
+React components that pass multiple sets of children can be migrated to an Astro component using [named slots](/en/core-concepts/astro-components/#named-slots).
+
+See more about [specific `<slot />` usage in Astro](/en/core-concepts/astro-components/#slots).
+
+### Next Data Fetching to Astro
+
+Convert any instances of `getStaticProps()` to `Astro.glob()` or `getCollections()` to access data from other files in your project source. To [fetch remote data](/en/guides/data-fetching/), use `fetch()`.
+
+These data requests are made in the frontmatter of the Astro component and use top-level await.
+
+```astro title="src/pages/index.astro"
+---
+import { getCollection } from 'astro:content';
+
+// Get all `src/content/blog/` entries
+const allBlogPosts = await getCollection('blog');
+
+// Get all `src/pages/posts/` entries
+const allPosts = await Astro.glob('../pages/posts/*.md');
+
+const response = await fetch('https://randomuser.me/api/');
+const data = await response.json();
+const randomUser = data.results[0];
+---
+```
+
+See more about [local files imports with `Astro.glob()`](/en/guides/imports/#astroglob) or [querying using the Collections API]([fetching remote data](/en/guides/data-fetching/)).
+
+### Next Styling to Astro
+
+Convert any inline style objects (`style={{fontWeight: "bold"}}`) to inline HTML style attributes (`style="font-weight:bold;"`). Or, use an Astro `<style>` tag for scoped CSS styles.
+
+```jsx title="src/components/Card.jsx"
+<div style={{backgroundColor: `#f4f4f4`, padding: `1em`}}>{message}</div>
+```
+
+
+```astro title="src/components/Card.astro"
+<div style="background-color: #f4f4f4; padding: 1em;">{message}</div>
+```
+
+You may also need to replace any CSS-in-JS libraries (e.g. styled-components) with other available CSS options in Astro.
```

Comment:
Worth linking to [this helpful issue](https://github.com/withastro/astro/issues/4432)?